options_test.go

```diff
@@ -326,3 +326,33 @@ func TestGCPHealthcheck(t *testing.T) {
 	o.GCPHealthChecks = true
 	assert.Equal(t, nil, o.Validate())
 }
+
+func TestRealClientIPHeader(t *testing.T) {
```

Comment:
Can we add a comment to this test to outline what we're asserting.

Reply:
Yes of course!

Reply:
I added the comments below for each case, what do you think?